charts/kubeclarity/templates/service.yaml

```diff
@@ -3,8 +3,8 @@ kind: Service
 metadata:
   name: {{ include "kubeclarity.name" . }}
   namespace: '{{ .Release.Namespace }}'
-  labels:
-    {{ include "kubeclarity.labels" . }}
+  labels: {{ include "kubeclarity.labels" . | nindent 4 }}
+    app: {{ include "kubeclarity.name" . }}
```

Comment:
why we need `app: {{ include "kubeclarity.name" . }}` only on the service?
we can just add it to `kubeclarity.labels`, WDYT?

Reply:
good idea